data/json/items/ammo/762x25.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "id": "762_25",
+    "type": "AMMO",
+    "name": "7.62x25mm JHP",
+    "name_plural": "7.62x25mm JHP",
+    "description": "A commercial version of the 7.62x25mm cartridge created for the armed forces of Soviet Russia.  It was derived from the 7.63x25mm cartridge used by the C96 pistol.",
+    "weight": 6,
```

Comment:
I am having a hard time finding cartridge weight for the 7.62x25mm. I would look at the 7.62x25mm data at benning.army.mil but I don't think i'm allowed to access it.
iaaforums.org said the bullet weight of the Type P/Type/ 51/Type 64 is approx. 7.7 and the cartridge weight is 12-13 grams.

The 116 gr 9mm JHP bullet is lighter than the 85 grain 7.62x25 JHP.